src/collector/facet_collector.rs

```diff
@@ -472,7 +472,7 @@ mod tests {
     #[test]
     fn test_facet_collector_drilldown() -> crate::Result<()> {
         let mut schema_builder = Schema::builder();
-        let facet_field = schema_builder.add_facet_field("facet", INDEXED);
```

Comment:
we could think about keeping INDEXED which would equal FacetOptions::default()?

Reply:
Yes it might help backward compatibility.